doc/users/next_whats_new/3d_axis_positions.rst

```diff
@@ -0,0 +1,19 @@
+Specify ticks and axis label positions for 3D plots
+---------------------------------------------------
+
+You can now specify the positions of ticks and axis labels for 3D plots.
+
+.. plot::
+   :include-source:
+
+   import matplotlib.pyplot as plt
+
+   positions = ['default', 'upper', 'lower', 'both', 'none']
+   fig, axs = plt.subplots(1, len(positions), figsize=(20, 4),
```

Comment:
The content width in docs is on 7in wide, so a 20in figure will more than 2x squashed down. Can we make this two rows? Or 2x2 like the test image, and just note which one is default in the title.

Reply:
Yeah, made this two rows with a (12, 8) figsize. Hard to go much smaller than that without axis labels overlapping eachother.